lib/parallel_tests/test/runner.rb

```diff
@@ -162,7 +162,7 @@ def capture_output(out, silence)
         end
 
         def sort_by_runtime(tests, runtimes, options={})
-          allowed_missing = options[:allowed_missing] || 1.0
+          allowed_missing = options[:allowed_missing] || 0.5
```

Comment:
this 1.0 was there on purpose for the nil case ... I think we'll have to use allow_missing in both cases ... maybe put the default somewhere higher up